src/main/java/com/parasoft/findings/jenkins/coverage/ParasoftCoverageRecorder.java

```diff
@@ -246,16 +257,22 @@ public String defaultPattern() {
     static class CoverageConversionResult {
         private final String coberturaPattern;
         private final Set<String> generatedCoverageBuildDirs;
+        private final String sourceCodeEncoding;
 
-        public CoverageConversionResult(String coberturaPattern, Set<String> generatedCoverageBuildDirs) {
+        public CoverageConversionResult(String coberturaPattern, Set<String> generatedCoverageBuildDirs, String sourceCodeEncoding) {
```

Comment:
The `sourceCodeEncoding` is not related to  `CoverageConversionResult`(which is used to store the result of convert Parasoft report to Cobertura report), we do not need it here.